bokeh/core/json_encoder.py

```diff
@@ -49,7 +49,7 @@
 pd = import_optional('pandas')
 rd = import_optional("dateutil.relativedelta")
 
-NP_EPOCH = np.datetime64('1970-01-01T00:00:00Z')
+NP_EPOCH = np.datetime64(0, 's')
```

Comment:
The 's' was for consistency with the current string parsed result, but yeah 'ms' makes more sense.  Might be a good idea to make this _NP_EPOCH while we're at it?